server/api/v1/workspace_policy_service.go

```diff
@@ -0,0 +1,241 @@
+package v1
```

Comment:
Just call this API org_policy_service.go?

Reply:
How about workspace_policy_service?